example-get-started/generate.sh

```diff
@@ -301,16 +301,26 @@ EOF
   if [ $OPT_DVC_TRACKED_METRICS == "true" ]; then
     dvc stage add -n evaluate \
       -d src/evaluate.py -d model.pkl -d data/features \
-      -m eval/live/metrics.json -o eval/live/plots \
+      -o eval/live/metrics.json -o eval/live/plots \
```

Comment:
it should be `-O` by default I think (`cache: false`) - just to double check

Reply:
Right, here because `$OPT_DVC_TRACKED_METRICS == "true"`, we use `-o`. Otherwise, we can use `-O` or drop them completely (see above).